tools/versioner/cmd/root.go

```
@@ -77,16 +88,13 @@ func Execute() {
 
 // wrapper for cobra, prints tag to stdout
 func theCommand(args []string) error {
-	tag, err := theCommandImpl(args)
-	if err == nil {
-		fmt.Printf("tag: %s\n", tag)
-	}
+	_, err := theCommandImpl(args)
 	return err
 }
 
 // does the actual work
 func theCommandImpl(args []string) (string, error) {
-	stage := filepath.Clean(args[0])
+	stage, _ := filepath.Abs(args[0])
```

Don't discard potential error.

---

Done